modules/SofaTopologyMapping/Hexa2QuadTopologicalMapping.cpp

```diff
@@ -101,10 +106,8 @@ void Hexa2QuadTopologicalMapping::init()
         return;
     }
 
-    QuadSetTopologyContainer *to_tstc;
-    toModel->getContext()->get(to_tstc);
```

Comment:
Yes, toModel is a BaseMeshTopology, so it could be something else than a QuadSetTopologyContainer. I'm not sure if casting it to a QuadSetTopologyContainer is really necessary here... but at least this PR keeps the "toModel must be a QuadSetTopologyContainer" requirement.